auslib/test/blobs/test_gmp.py

```diff
@@ -59,6 +59,51 @@ def setUp(self):
 }
 """)
 
+    def testValidateHashValue(self):
+        blob = GMPBlobV1()
+        blob.whitelistedDomains = {'boring.com': ('gg',), }
+        blob.loadJSON("""
+{
+    "name": "validName",
+    "schema_version": 1000,
+    "hashFunction": "SHA512",
+    "vendors": {
+        "a": {
+            "version": "5",
+            "platforms": {
+                "default": {
+                    "filesize": 20,
+                    "hashValue": "50",
+                    "fileUrl": "http://boring.com/bar"
+                }
+            }
+        }
+    }
+}
+""")
+        self.assertRaises(Exception, blob.validate, 'gg', self.whitelistedDomains)
```

Comment:
It would be better to catch ValueError + check the message here with assertRaisesRegexp, to make sure it's actually failing in the correct way.